contents/uix/uix.md

```diff
@@ -0,0 +1,47 @@
+# UI/UX
+
+Author: [Ang Shi Ya](https://github.com/AngShiYa)
+
+## Overview
+
+User Interface (UI) and User Experience (UX) are terms often used interchangeably. In fact, these two terms are quickly becoming dangerous buzzwords that are used inaccurately, creating massive confusion in the industry. A simple google search will show that UI Designer and UX Designer jobs are listed under similar titles and descriptions. 
+
+In addition to helping us to identify and apply for the right role, understanding the key differences between UI and UX helps us in using the right concept in our own applications. This document will address the difference between UI and UX as well as some of the techniques used in UI Design and UX Design.
```

Comment:
I reread this sentence and realised the "to" before "identify" is unnecessary and breaks the flow of the sentence a little. Not a major problem though if you don't want to remove it!